### DISCUSSION TITLE
September 6, 2024 meeting

### DISCUSSION THREAD

**Discussion by azukov** [PyORBIT-Collaboration/PyORBIT3#25] [General]
The agenda for now is:

- Closing meson build? (Sasha)

-  Any other open tickets?

-  Any new development on py2 side that needs transferring

- Round table and AoB

**Comment by juanfem** [comment]
Summary of the discussions:
- Yngve became Control Room Operations group, expecting less participation in this project. Daniel Noll will try to contribute. I'll take care of setting up this meeting from now on.
- Andrei reports that they used PyORBIT to analyze phase scans in SCL. Results were expected to be more accurate than using OpenXAL, but in the end they were similar in accuracy.
- We decided to halt developments in the Python 2 version, updating the readme file to inform users to use the Python 3 version.
- Discussion about documentation. Proposal: start by making sure the functions used in the main examples are documented.
- Brandon will give a presentation about the virtual accelerator in one of the next meetings.
- Next meeting will be October 11, same time (15:00 UTC+1, 9:00 UTC-4)
